src/app/router/connectorRoutes.tsx

```diff
@@ -13,9 +13,11 @@ import NewWallet from "@screens/connectors/NewWallet";
 import i18n from "~/i18n/i18nConfig";
 import { translationI18nNamespace } from "~/i18n/namespaces";
 
+import ConnectCommando from "../screens/connectors/ConnectCommando";
 import alby from "/static/assets/icons/alby.png";
 import btcpay from "/static/assets/icons/btcpay.svg";
 import citadel from "/static/assets/icons/citadel.png";
+import core_ln from "/static/assets/icons/core_ln.svg";
```

Comment:
should we mark this with some `beta` badge?

Reply:
did we ever do this with any other connector? Why should we do it here then?

Reply:
that's what I am asking. 
we did not have any other connector added at this stage that Alby is in now.  

LND for example was the first one used for months before fore even release Alby. 
Eclair would have been one, but that was also added very early on. so the sate of Alby was a very different. 

And this connector relies on a very new library a new websocket proxy, etc. so it is also different and not only a simple HTTP call.

Reply:
I personally don't feel a need to add it.

Reply:
So you're confident enough that this (and lnmessage) works reliable for the various CLN installations?

Reply:
Yes. And to add to that I am also confident that there is nobody that will care if there is a "beta" badge here or not.